tests/tests.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+from unittest import TestCase
+import json
+
+from openfisca_web_api.app import create_app
+
+tester = create_app('openfisca_dummy_country').test_client()
+
+
+class ParametersRoute(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.response = tester.get('/parameters')
+
+    def test_return_code(self):
+        self.assertEqual(self.response.status_code, 200)
+
+    def test_headers(self):
+        headers = self.response.headers
+        self.assertEqual(headers.get('Country-Package'), 'openfisca-dummy-country')
+        self.assertEqual(headers.get('Country-Package-Version'), '0.1.1')
```

Comment:
Try to make test functions atomic:

```python
def test_package_name_header(self):
    self.assertEqual(self.response.headers.get('Country-Package'), 'openfisca-dummy-country'))

def test_package_version_header(self):
    self.assertEqual(self.response.headers.get('Country-Package-Version'), '0.1.1'))
```

…will yield much more informative failure messages and will scale more.